packages/lesswrong/server/scripts/mergeAccounts.js

```diff
@@ -147,3 +157,23 @@ Vulcan.mergeAccounts = async (sourceUserId, targetUserId) => {
     validate: false
   })
 }
+
+
+async function recomputeKarma(userId) {
+  const user = await Users.findOne({_id: userId})
+  if (!user) throw Error("Can't find user")
+  const allTargetVotes = await Votes.find({
```

Comment:
This is only partially indexed (the closest is missing `legacy`), so there's a risk that this will table-scan. But one table-scan per manual account-merge operation might actually just be fine.